sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/AzureMonitorTraceExporter.cs

```diff
@@ -17,6 +15,7 @@ public class AzureMonitorTraceExporter : BaseExporter<Activity>
         private readonly ITransmitter Transmitter;
         private readonly AzureMonitorExporterOptions options;
         private readonly string instrumentationKey;
+        private const int maxExportBatchSize = 512;
```

Comment:
We might get a question, why 512 and not some other value. It's better to have a reason is summary of PR why batch size is 512.